rpcs3/CMakeLists.txt

```diff
@@ -282,7 +284,31 @@ if(USE_SYSTEM_FFMPEG)
 	include_directories("${FFMPEG_INCLUDE_DIR}")
 else()
 	message("-- RPCS3: using builtin ffmpeg")
-	include_directories("${RPCS3_SRC_DIR}/../3rdparty/ffmpeg/${PLATFORM_ARCH}/include")
+	if(MSVC OR NOT WIN32)
```

Comment:
I think it would be better if this was upstreamed to https://github.com/hrydgard/ppsspp-ffmpeg. In the meantime you can add `-DUSE_SYSTEM_FFMPEG=on` to the build instructions.